node/templates/bird.cfg.template

```diff
@@ -1,13 +1,15 @@
-# Generated by confd {{datetime}}
+# Generated by confd
 router id {{getenv "IP"}};
 log "/var/log/calico/bird.log" all;
 
 filter calico_pools {
-{{range gets "/calico/v1/ipam/v4/pool/*"}}
+{{if exists "/ipam/v4/pool"}}
```

Comment:
Is it possible to add a comment in the template without it going into the actual bird config?